docs/csharp/nullable-references.md

```diff
@@ -151,9 +153,9 @@ The new features that protect against throwing a <xref:System.NullReferenceExcep
 - The meaning of the `class` constraint in generics changed to mean a non-nullable reference type.
 - New warnings are generated because of these new rules.
 
-You must explicitly opt in to use these features in your projects. That provides a migration path and preserves backwards compatibility. Nullable contexts enable fine-grained control for how the compiler interprets reference type variables. The **nullable annotation context** determines the compiler's behavior. There are four values for the **nullable annotation context**:
+You must explicitly opt in to use these features in your existing projects. That provides a migration path and preserves backwards compatibility. Nullable contexts enable fine-grained control for how the compiler interprets reference type variables. The **nullable annotation context** determines the compiler's behavior. There are four values for the **nullable annotation context**:
```

Comment:
```suggestion
You must explicitly opt-in to use these features in your existing projects. That provides a migration path and preserves backwards compatibility. Nullable contexts enable fine-grained control for how the compiler interprets reference type variables. The **nullable annotation context** determines the compiler's behavior. There are four values for the **nullable annotation context**:
```

Reply:
I'm going to leave this as-is based on https://www.merriam-webster.com/dictionary/opt%20in.